distributed/actor.py

```diff
@@ -119,11 +119,14 @@ def _worker_rpc(self):
                 return ProxyRPC(self._client.scheduler, self._address)
 
     @property
-    def _asynchronous(self):
+    def _asynchronous(self) -> bool:
         if self._client:
             return self._client.asynchronous
-        else:
-            return threading.get_ident() == self._worker.thread_id
+        try:
+            asyncio_loop = asyncio.get_running_loop()
+        except RuntimeError:
+            return False
+        return self._worker.loop.asyncio_loop is asyncio_loop
```

Comment:
this wasn't covered either that's odd

Reply:
This might be worthwhile simplifying then, but that's out of scope for this PR.